crates/macro/src/html_tree/html_node.rs

```diff
@@ -18,7 +35,13 @@ impl Parse for HtmlNode {
             }
             Node::Literal(lit)
         } else {
-            Node::Raw(input.parse()?)
+            let mut stream = TokenStream::new();
+            let content: Vec<Stmt> = Block::parse_within(input).unwrap();
+            for mut stmt in content {
+                HtmlInnerModifier.visit_stmt_mut(&mut stmt);
+                stream.extend(quote! {#stmt})
+            }
+            Node::Raw(stream)
```

Comment:
This doesn't work because of the parsing is of a collection of statements. Since the braces were stripped before hand. This is the only way I was able to get it to run. Later when I remove the HtmlIterator and such I will rework this to be the way you suggest.